### ISSUE TITLE
Accessibility bug in the alert message in the Choose a location page within the cheetah MVP flow

### ISSUE THREAD

**Issue by jenniemc** [department-of-veterans-affairs/va.gov-team#21867]
Discovered in #21342

**Choose a location page:**

- Heading order is invalid on the alert messages. It currently goes from h1 to h3.
- Expected behavior on the heading of the alert box, "We found one VA location for you"  should be h2

Steps to reproduce:

1. Go into `src/applications/vaos/services/mocks/var/direct_booking_eligibility_criteria.json` remove `patientHistoryRequired` from `covid` typeof where clinic id=983QA
2. Select `No` on the `Have you received a Covid-19 vaccine` page
3. "We found one VA location for you"  should be set to h2

![image.png](https://images.zenhubusercontent.com/5d6463ed1325bafe0674800b/d0b8560d-8ca6-468d-8f5c-7d6e1b9d7666)

Steps to reproduce:

1. Go into `src/applications/vaos/services/mocks/var/direct_booking_eligibility_criteria.json` remove `patientHistoryRequired` from `covid` typeof where clinic id=983
2. Select `No` on the `Have you received a Covid-19 vaccine` page
3. "We found one VA location for you"  should be set to h2

![image.png](https://images.zenhubusercontent.com/5d6463ed1325bafe0674800b/29db96e0-68da-4766-b065-6eba4dbd70a6)

**Comment by jenniemc** [comment]
@laurenernest, I don't think this is a design problem but is a coding issue that the FE can fix.

**Comment by laurenernest** [comment]
This one is tied to a larger design pattern issue

**Comment by laurenernest** [comment]
Closing this ticket as waived. Existing pattern for the alert component that's elsewhere in VAOS. We'll be addressing this with broader alert-related changes.

**Comment by jbalboni** [comment]
@laurenernest This issue is because we have the design system alert component misconfigured on this page. It's minor, but something we can and should fix.

**Comment by 1Copenut** [comment]
Thanks @jbalboni I appreciate you fixing this one. It's an important item for screen reader users.
